bundles/org.eclipse.orion.client.git/web/orion/git/widgets/ApplyPatchDialog.js

```diff
@@ -21,11 +21,13 @@ define([ 'i18n!git/nls/gitmessages', 'orion/webui/dialog', 'orion/xsrfUtils', 'o
 
 	ApplyPatchDialog.prototype.TEMPLATE =
 
+	'<span role="radiogroup" aria-label="${Patch Location}>"' +
```

Comment:
If possible, I would use a div here and not a span. Technically, the [content model for span](http://w3c.github.io/html/textlevel-semantics.html#the-span-element) is to only have [phrasing content](http://w3c.github.io/html/dom.html#phrasing-content-2), which does not include div.

See the Note in the definition of "phrasing content" that says:
> Most elements that are categorized as phrasing content can only contain elements that are themselves categorized as phrasing content, not any flow content.

Reply:
done